src/recorder.js

```diff
@@ -1,6 +1,9 @@
 import { dimensionsOf } from './util.js';
 
 export default class Recorder {
+
+  #recorder
```

Comment:
Semicolon and tiny doc comment missing.

Reply:
There are no semicolons after the method definitions, either, so I was thinking: No semicolons after field declarations/definitions. So that assumption is wrong, yes?

As for the comment: Do you insist? ;P I don't think this being made private suddenly necessitates a comment. It had none before, when it was just defined in place, and you know how I despise comments like `/** The recorder */`. :thinking:

Reply:
yeeee ok it's fine